VagrantProvision.sh

```diff
@@ -244,21 +334,22 @@ PATH=$HOOT_HOME/bin:$PATH
 EOT
 fi
 
-if ! grep -i --quiet 'umask 002' /etc/default/tomcat6; then
-echo "Changing Tomcat umask to group write..."
+# Was 002, changing it to 000 to get rid of errors
+if ! grep -i --quiet 'umask 000' /etc/default/tomcat6; then
+echo "### Changing Tomcat umask to group write..."
 sudo bash -c "cat >> /etc/default/tomcat6" <<EOT
 # Set tomcat6 umask to group write because all files in shared folder are owned by vagrant
-umask 002
+umask 000
```

Comment:
Ah, I recall now that the group owner option down below doesn't work with nfs.

I don't see any additions to the Vagrantfile that would cause it to use rsync if Virtualbox is the provider.  I think that will continue to be the default for github users testing hoot out.